Automattic/jetpack
# P2 admin menu: update fxn signature to fix PHP warnings

#### Changes proposed in this Pull Request:

* D71956-code has updated the function signature of `add_woocommerce_installation_menu()` to add an optional parameter. Since we are overriding this function for P2 sites, we must also update the overriding function to silence PHP warnings about incompatible function declarations.

#### Jetpack product discussion


No product changes involved.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Apply these changes to your wpcom sandbox and visit a sandboxed P2 site.
* Check the PHP logs and verify that the warnings about incompatible function declarations are gone.
